sale_order_import_invoice2data/wizard/sale_order_import_pdf.py

```diff
@@ -0,0 +1,178 @@
+# -*- coding: utf-8 -*-
+# © 2016 Sunflower IT (http://sunflowerweb.nl)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, api, tools, _
+from openerp.exceptions import Warning as UserError
+from openerp.tools import float_compare
+from tempfile import mkstemp
+import base64
+import unicodecsv
+import os
+import re
+import pkg_resources
+import logging
```

Comment:
order your imports according to the [conventions](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports)